src/components/layout/Card.tsx

```diff
@@ -85,7 +94,9 @@ const Card = (props: PropsWithChildren<Props>) => {
             {content}
           </button>
         ) : (
-          <article className={cardClassName}>{content}</article>
+          <article className={cx(cardClassName, inputCard ? "inputCard" : "")}>
```

Comment:
Here, since it is inside cx() you can replace `cx(cardClassName, inputCard ? "inputCard" : "")` with `cx(cardClassName, { inputCard })`.
Minor thing but it's a bit cleaner